pkg/services/mssql/plan_details.go

```diff
@@ -53,7 +53,23 @@ func (d dtuPlanDetails) getUpdateSchema() service.InputParametersSchema {
 }
 
 func (d dtuPlanDetails) getProvisionSchema() service.InputParametersSchema {
-	return d.getUpdateSchema()
+	ips := service.InputParametersSchema{
+		PropertySchemas: map[string]service.PropertySchema{},
+	}
+	if d.includeDBMS {
+		ips = getDBMSCommonProvisionParamSchema()
+	}
+	// Basic tier is constrained to just 5 DTUs, so don't present this as an
+	// option
+	if len(d.allowedDTUs) > 0 {
+		ips.PropertySchemas["dtus"] = &service.IntPropertySchema{
+			AllowedValues: d.allowedDTUs,
+			DefaultValue:  ptr.ToInt64(d.defaultDTUs),
+			Description: "DTUs are a bundled measure of compute, " +
+				"storage, and IO resources.",
+		}
+	}
+	return ips
```

Comment:
Could probably do some DRY refactor here, but I feel like explicitly repeating this is a little more readable.